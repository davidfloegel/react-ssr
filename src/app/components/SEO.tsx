import React from 'react';
import Helmet from 'react-helmet';

interface Props {
  title: string;
}

const SEO: React.SFC<Props> = ({ title }) => (
  <Helmet>
    <title>{title}</title>
  </Helmet>
);

export default SEO;
