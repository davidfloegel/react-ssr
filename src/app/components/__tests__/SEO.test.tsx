import React from 'react';

import SEO from 'components/SEO';

describe('SEO Component', () => {
  test('it renders with a title', () => {
    expect(<SEO title="Test TItle" />).toMatchSnapshot();
  });
});
