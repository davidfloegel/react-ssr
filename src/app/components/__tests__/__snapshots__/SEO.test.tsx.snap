// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO Component it renders with a title 1`] = `
<SEO
  title="Test TItle"
/>
`;
